src/map.rs

```diff
@@ -3377,7 +3377,6 @@ mod test_map {
 
         let mut m = HashMap::new();
 
-        // FIXME: https://github.com/rust-lang/miri/issues/653
         let mut rng = {
             let seed = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16];
             SmallRng::from_seed(seed)
```

Comment:
This issue has been closed, meaning `getrandom` now works on Miri. This is still not host randomness by default; see the [Miri README](https://github.com/rust-lang/miri/#miri--z-flags-and-environment-variables). But wouldn't it be better to keep this test deterministic? That's why I didn't change the code.